libraries/bxdf/gltf_pbr.mtlx

```diff
@@ -25,7 +25,7 @@
     <input name="emissive" type="color3" value="0, 0, 0" uimin="0, 0, 0" uimax="1, 1, 1" uiname="Emissive" uifolder="Emission" />
     <input name="emissive_strength" uniform="true" type="float" value="1" uimin="0" uiname="Emissive Strength" uifolder="Emission" />
     <input name="thickness" uniform="false" type="float" value="0" uimin="0" uiname="Thickness" uifolder="Volume" />
-    <input name="attenuation_distance" uniform="true" type="float" uimin="0" uiname="Attenuation Distance" uifolder="Volume" />
+    <input name="attenuation_distance" uniform="true" type="float" value="1e+36" uimin="0" uiname="Attenuation Distance" uifolder="Volume" />
```

Comment:
These additional updates to the standard data libraries look like good proposals, but I'd recommend making them as separate, clearly-focused pull requests, since they'll have a different set of users interested in reviewing them.

Specifically for this update to the glTF PBR shading interface, we'll want to bring Khronos into the conversation, to make sure that we're choosing the best, arbitrarily-high value for this default.

Reply:
I'll see what is separable. I ran into issues with validation due to a number of these.